src/css/ValidationTypes.js

```diff
@@ -210,6 +210,13 @@ var ValidationTypes = {
 
         "<flex-wrap>": function(part){
             return ValidationTypes.isLiteral(part, "nowrap | wrap | wrap-reverse");
+        },
+        
+        "<feature-tag-value>": function(part){
+            if (part.type == "function" && /^[A-Z0-9]{4}$/i.test(part)){
```

Comment:
Am I correct in understanding that you mean I can just do `return if (part.type == "function" && /^[A-Z0-9]{4}$/i.test(part))` or did I misunderstand?